cmd/ebitenmobile/gobind.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 The Ebiten Authors
+// Copyright 2022 The Ebiten Authors
```

Comment:
Do no update years in the existing files.

Reply:
Ok